tmva/pymva/inc/TMVA/MethodPyAdaBoost.h

```diff
@@ -19,24 +19,23 @@
 //                                                                      //
 // MethodPyAdaBoost                                                     //
 //                                                                      //
-//                                                                      //
 //////////////////////////////////////////////////////////////////////////
 
+#ifndef ROOT_TMVA_PyMethodBase
```

Comment:
Could you remove these redundant header guards? We just removed all of them from the current repo in #341